### ISSUE TITLE
App overlay overflows to task bar

### ISSUE THREAD

**Issue by fryorcraken** [status-im/status-react#20833]
# Bug Report

## Problem 
The app overflows to the top task bar in android, meaning button at the top in the app are not reachable.

Note it doesnt always happen.

#### Expected behavior
Lower button position for top buttons.

Sometimes it works:

![Screenshot_20240721-132552.png](https://github.com/user-attachments/assets/ed378cce-3ecf-420c-a582-f542786a1e20)

![Screenshot_20240721-132540.png](https://github.com/user-attachments/assets/7b990b0b-6489-4c4d-ac3a-84995721ed2f)

#### Actual behavior

![Screenshot_20240721-131816.png](https://github.com/user-attachments/assets/114983ed-04d5-4c40-8b26-ffd36d75cda4)

![Screenshot_20240721-132309.png](https://github.com/user-attachments/assets/bd23548f-c68c-4200-89de-a33cb341ef53)

https://drive.proton.me/urls/Z1CTGEYTCM#qZ7YK4lmulT2

### Reproduction

1. 

### Additional Information

- Status version: 97b19c54401b9fffc003f0b79feff83e21f55851
- Operating System: Android, Grapheneos, Pixel 7

[comment]: # (Please, add logs/notes if necessary)

**Comment by ilmotta** [comment]
@churik I've seen a few recent issues being reported by users of GrapheneOS. Does the QA team have a real device with GrapheneOS to attempt to reproduce issues in this OS?

If we can reproduce the issue in other more common OSes then it's all good, but my concern is: what do we do if we cannot reproduce a bug reported by a GrapheneOS user because we don't have a device to test. Would the QA team have capacity to cater for GrapheneOS?

What are your thoughts?

**Comment by churik** [comment]
To be honest, it is just too many of them.
Xiaomi and Huawei have their own MIUI shells (which we have), Pixel has Graphene OS, Samsung might have their own shells as well. 

I don't think that GrapheneOS is the most popular ones. Would be nice to have some stats, as our selection of devices was based on most popular ones on the time of releasing v1 + variety that we can provide within the team on Android devices (chinese models / samsung / google + different OS combinations). If you think it makes sense, we can install Graphene OS of one of our devices, but how we will prioritize these issues - also not clear.

This particular one is purely UI from my perspective, and to be honest we have issues that are a way more serious than this one  (which are device/OS specific), i.e. https://github.com/status-im/status-mobile/issues/18173 or when you're trying to launch app on devices with small resolution, you may see the overlapping elements or in general wrong image recizing, i.e.

![image](https://github.com/user-attachments/assets/b40a6d4f-293c-4441-9ce8-402699c1024b)

So bottom line from me is: we can do this, if you think it makes sense. For now I'd say we have a way more issues in general handling variety of devices and we should draw the line somewhere. 
@ilmotta

**Comment by ilmotta** [comment]
I agree with your points @churik. As of 2024-09-02, the top 10 devices from the active Android installed base is:

1. Samsung Galaxy S23 Ultra1
2. Samsung Galaxy A51
3. Samsung Galaxy S24 Ultra
4. Samsung Galaxy A50
5. Samsung Galaxy S22 Ultra
6. Samsung Galaxy A21s
7. Huawei Y9 Prime 2019
8. Samsung Galaxy A10
9. Samsung Galaxy A31
10. Samsung Galaxy A71

Most other devices further down the list are Samsung, and only a few from Xiami. I don't see a way to precisely breakdown by OS, this feature doesn't exist in Play Console. As far as I know, GrapheneOS basically works with Pixel devices, not Samsung devices (excluding users who possibly rooted devices).

> So bottom line from me is: we can do this, if you think it makes sense. For now I'd say we have a way more issues in general handling variety of devices and we should draw the line somewhere.

I think we draw the line based on data from Play Console and also if the bug is critical or not. Right now it doesn't make sense for the mobile team to investigate and fix bugs for GrapheneOS **unless they are impeding the user in a more severe way**.

@fryorcraken we hope this point of view is understandable. Please, continue reporting bugs for GrapheneOS because such issues can be helpful in the future.

**Comment by ilmotta** [comment]
FYI: @fryorcraken We are considering increasing support for Graphene OS due to increased user feedback. **Not testing on a continuous basis**, but at least before a release to make sure critical bugs don't get to Graphene OS users. cc @churik

**Comment by ulisesmac** [comment]
Related to safe-areas
